### PR TITLE
recipe for theseus

### PR DESCRIPTION
**Pull request by Tiggha**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jaimergp** [comment]
I hereby confirm I will maintain this recipe!

File: recipes/theseus/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (synapticarbors) [inline recipes/theseus/LICENSE.txt:1]: When I look at the tarball, it looks like the license file is included there as the `COPYING` file, so I think you can remove this file and use `license_file: COPYING` in the `meta.yaml`.

File: recipes/theseus/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://theobald.brandeis.edu/theseus/src/{{ name }}_{{ version }}.tar.gz
+  sha256: 3cfd4f906717f9cb8e77f689fc97059b32df355c0696077034ea99a485e3f2fe
+
+build:
+  number: 0
```
Comment (synapticarbors) [inline recipes/theseus/meta.yaml:13]: If you're only supplying a `build.sh` and therefore implying that a Windows build is not supported, please add:

```
skip: True  # [win]
```

under `number` in the `build` section.

Comment (synapticarbors) [reply recipes/theseus/meta.yaml:13]: a windows build would require a `bld.bat` file equivalent to the `build.sh`

**Comment by synapticarbors** [comment]
Closing in favor of https://github.com/conda-forge/staged-recipes/pull/11296

**Pull request closed without merging**